### PR TITLE
Some fixes for DGtal for dummies

### PR DESCRIPTION
**Pull request by JacquesOlivierLachaud**
# PR Description

Small fixes in format of some files. Add a few missing methods in Triangulated and Polygonal Surface. Add colormaps to shortcuts and some fixes in TickedColorMap.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ ] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/doc/tutorials/moduleShortcuts.dox [hunk 1]
```diff
@@ -443,7 +443,7 @@ Shortcuts::Point is Z3i::KSpace::Point.
 -# Utilities
   - Shortcuts::parametersUtilities: parameters related to colormaps.
   - Shortcuts::getColorMap: returns the specified colormap.
-  - Shortcuts::getTicsColorMap: returns the specified colormap used for tics.
+  - Shortcuts::getZeroTicColorMap: returns the specified colormap with a tic around zero.
```
Comment (dcoeurjo) [inline src/DGtal/doc/tutorials/moduleShortcuts.dox]:
```suggestion
  - Shortcuts::getZeroTickedColorMap: returns the specified colormap with a tick around zero.
```

(+update the code)

File: src/DGtal/shapes/PolygonalSurface.h [hunk 2]
```diff
@@ -502,6 +509,14 @@ namespace DGtal
     */
     VertexRange verticesAroundFace( const Face & f ) const;
 
+    /**
+       @param f any valid face on the surface. (open or closed ).
+
+       @return the sequence of arcs that touches this face in their
```
Comment (dcoeurjo) [inline src/DGtal/shapes/PolygonalSurface.h]:
```suggestion
      @note By construction, each @e anArc returned by the method on a face @a f is such that `poly.faceAroundArc(anArc) == f`.

       @return the sequence of arcs that touches this face in their
```

**Comment by dcoeurjo** [comment]
Nope... works fine on my mac. 
It seems to be Travis related

**Comment by JacquesOlivierLachaud** [comment]
Was there a change in version of ImageMagick in the travis CI ?
This error seems completely unrelated to this PR.

**Comment by dcoeurjo** [comment]
travis is using the last (brew) version of imagemagick. Weird because on my mac, I have the exact same version and no problem for me

**Comment by dcoeurjo** [comment]
correct.. I see it as a warning also on my mac
we should deactivate the warning (using pragmas around the header files inclusion)

**Comment by dcoeurjo** [comment]
all good, thx, merging

**Pull request merged**